src/config.cpp

```diff
@@ -519,6 +519,11 @@ void parse_program_options(int argc, const char** argv)
         config.disassemble = false;
 #endif
     }
+    if (metric_frequency == 0 || metric_count == 0)
```

Comment:
Hold on, one of them should always be zero, the other nonzero, right?

Reply:
yeah misinterpreted a line where we initialize them